pkg/command/plugin_test.go

```diff
@@ -198,3 +200,126 @@ func TestDeletePlugin(t *testing.T) {
 		os.Unsetenv("TEST_CUSTOM_CATALOG_CACHE_DIR")
 	}
 }
+
+func TestInstallPlugin(t *testing.T) {
+	tests := []struct {
+		test               string
+		centralRepoFeature string
+		args               []string
+		expectedErrorMsg   string
+		expectedFailure    bool
+	}{
+		{
+			test:               "no 'all' option with central repo",
+			centralRepoFeature: "true",
+			args:               []string{"plugin", "install", "all"},
+			expectedFailure:    true,
+			expectedErrorMsg:   "the 'all' argument is not longer supported. You must install each plugin individually",
+		},
+		{
+			test:               "invalid target",
+			centralRepoFeature: "true",
+			args:               []string{"plugin", "install", "--target", "invalid", "myplugin"},
+			expectedFailure:    true,
+			expectedErrorMsg:   "invalid target specified. Please specify correct value of `--target` or `-t` flag from 'kubernetes/k8s/mission-control/tmc'",
+		},
```

Comment:
The above tests are only negative tests.
It is complicated to actually test installing a plugin from this part of the code, which I assume is why there was never a test for in this file.